docs/about.rst

```diff
@@ -2,13 +2,13 @@ About
 =====
 
 ----------------------------
-The EBI Metagenomics service
+The MGnify service
 ----------------------------
 
-EBI metagenomics is a freely available hub for the analysis and exploration of :term:`metagenomic`, :term:`metatranscriptomic`, :term:`amplicon` and :term:`assembly` data. The resource provides rich functional and taxonomic analyses of user-submitted sequences, as well as analysis of publicly available metagenomic datasets held within the European Nucleotide Archive (`ENA <https://www.ebi.ac.uk/ena>`_). 
+MGnify is a freely available hub for the analysis and exploration of :term:`metagenomic`, :term:`metatranscriptomic`, :term:`amplicon` and :term:`assembly` data. The resource provides rich functional and taxonomic analyses of user-submitted sequences, as well as analysis of publicly available metagenomic datasets held within the European Nucleotide Archive (`ENA <https://www.ebi.ac.uk/ena>`_).
```

Comment:
should we not indicate that Mgnify was initially EBI Metagenomics?

Reply:
from @almitchell say saomething like "the resource was previously known as EBI Metagenomics, but was rebranded to reflect its growing remit"